.travis.yml

```diff
@@ -20,12 +20,12 @@ matrix:
       env: TEST_CONFIG="phpunit.xml"
     - php: 5.4
       env: TEST_CONFIG="phpunit-integration-legacy.xml" DB="postgresql" DATABASE="pgsql://postgres@localhost/$DB_NAME"
-    - php: 5.4
-      env: SOLR_VERSION="4.10.4" TEST_CONFIG="phpunit-integration-legacy-solr.xml" CORES_SETUP="dedicated" SOLR_CORES="core0 core1 core2 core3 core4 core5 core6 core7" SOLR_CONFS="eZ/Publish/Core/Search/Solr/Content/Resources/schema.xml"
-    - php: 5.4
-      env: SOLR_VERSION="4.10.4" TEST_CONFIG="phpunit-integration-legacy-solr.xml" CORES_SETUP="shared" SOLR_CORES="core0 core1 core2 core3 core4" SOLR_CONFS="eZ/Publish/Core/Search/Solr/Content/Resources/schema.xml"
-    - php: 5.4
-      env: SOLR_VERSION="4.10.4" TEST_CONFIG="phpunit-integration-legacy-solr.xml" CORES_SETUP="single" SOLR_CONFS="eZ/Publish/Core/Search/Solr/Content/Resources/schema.xml"
+#    - php: 5.4
```

Comment:
can remove instead of comment out, not likely that we will move the code back in here.
As for testing upstream / downstream, no real feature for that on travis os we need to plan for using CI for such things.